src/main/java/org/opensearch/timeseries/ml/ModelManager.java

```diff
@@ -147,23 +145,22 @@ public <RCFDescriptor extends AnomalyDescriptor> IntermediateResultType score(
                 if (!modelState.getSamples().isEmpty()) {
                     for (Sample unProcessedSample : modelState.getSamples()) {
                         // we are sure that the process method will indeed return an instance of RCFDescriptor.
-                        rcfModel.process(unProcessedSample.getValueList(), unProcessedSample.getDataEndTime().getEpochSecond());
+                        double[] unProcessedPoint = unProcessedSample.getValueList();
+                        int[] missingIndices = DataUtil.generateMissingIndicesArray(unProcessedPoint);
```

Comment:
are these missing indices referring to the features that don't have values?

Reply:
yes.